setup.py

```diff
@@ -72,9 +72,9 @@
     'Flask-BabelEx>=0.9.3',
     'mistune>=0.7.2',
     'cchardet>=1.0.0',
-    'invenio-assets>=1.0.0a3',
-    'invenio-pidstore>=1.0.0a6',
-    'invenio-records-ui>=1.0.0a7',
+    'invenio-assets>=1.0.0b4',
+    'invenio-pidstore>=1.0.0b1',
+    'invenio-records-ui>=1.0.0a8',
```

Comment:
Yes, but I will merge anyway as we unfortunately had a production deployment so all previews are broken currently on zenodo. Mistake on our side but need fast track not this one.